src/main/scala/io/github/paoloboni/binance/fapi/FutureApi.scala

```diff
@@ -267,6 +267,43 @@ final case class FutureApi[F[_]: Logger](
     } yield response
   }
 
+  /** Gets order data.
+    *
+    * @param getOrder
+    *   the parameters required to find existing order
+    *
+    * @return
+    *   The id of the order created
+    */
+  def getOrder(getOrder: FutureGetOrderParams): F[FutureOrderCreateResponse] = {
```

Comment:
If you try to run the e2e test, you'll get back the following failure:
```
Attempt to decode value on failed cursor: DownField(cumQty)
```
This means that the data structure `FutureOrderCreateResponse` cannot be reused to decode this response.
I had a quick look here and I can see there's a slight difference. I think it's ok to have some duplication in this case and copy `FutureOrderCreateResponse` in a new `FutureOrderGetResponse` and change it accordingly to the api definition.